.github/tests/double_whitspace_test.py

```diff
@@ -0,0 +1,36 @@
+def find_bad_lines(file_content):
+    """
+    Find lines in the given file that have double whitespaces around commas or full stops.
+
+    :param file_content: A list of strings representing the contents of a file.
+    :return: A list of line numbers that contain double whitespaces around commas or full stops.
+    """
+    internal_bad_lines = []
+    for i, line in enumerate(file_content, start=1):
+        if ("  ." or ".  " or "  ," or ",  ") in line:
```

Comment:
Bei mir hatte das nicht funktioniert wie es war. Kann es an der Python Version liegen?
Alternativ hat bei mir das funktionier: 
```suggestion
        false_patterns = ["  .", ".  ", "  ,", ",  "]
        if any(x in line for x in false_patterns):
```

Reply:
Nicht funktioniert heißt hierbei, dass nur `  .` erkannt wurde und nichts anderes :-)